### ISSUE TITLE
On and off switches are not working

### ISSUE THREAD

**Issue by dignifiedquire** [tj/commander.js#108]
If I try 

``` js
program.command('start')
  .option('--auto-watch', 'Description')
  .option('--no-auto-watch', 'Description')
  .action(function(options) {
    console.log(options.autoWatch);
  });
```

I get the following output:

``` bash
$ node program.js start --auto-watch
false
$ node program.js start --no-auto-watch
false
$ node program.js start
true
```

That should not happen. I need the first one to be `true`. Also if I try to set a default value of `true` that won't work either.

**Comment by adius** [comment]
Any progress on this one?

**Comment by akdor1154** [comment]
Still hitting this in 2017..

**Comment by usmonster** [comment]
I have an open PR that should fix this: #795. Please review and comment!

**Comment by shadowspawn** [comment]
This issue will be resolved when v3.0.0 is released. Available now as a prerelease. See #1001

**Comment by shadowspawn** [comment]
Support for combined `--foo` and `--no-foo` has shipped in v3 (thanks @usmonster): https://github.com/tj/commander.js/releases/tag/v3.0.0

Closing second oldest open issue. Woop!
